.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,5 +27,6 @@ assignees: ''
  - OS:
  - Web browser:
  - Python version:
+ - Label Sleuth version (as appear in the bottom left of the UI):
```

Comment:
```suggestion
 - Label Sleuth version (as it appears on the bottom left of the UI):
```

Reply:
Thanks @alonh for asking for the version on the bug report template. It should help us with bug reports! Change looks good; I just made a minor change to phrasing (see suggestion).